sdk/communication/Azure.Communication.Administration/tests/samples/Sample2_PhoneNumberAdministrationClient.cs

```diff
@@ -103,14 +107,18 @@ await foreach (var phoneNumber in acquiredPhoneNumbers)
             #region Snippet:ReleasePhoneNumbersAsync
             //@@var acquiredPhoneNumber = "<acquired_phone_number>";
             var releaseOperation = client.StartReleasePhoneNumber(new PhoneNumberIdentifier(acquiredPhoneNumber));
-            await releaseOperation.WaitForCompletionAsync();
+            //@@ await releaseOperation.WaitForCompletionAsync();
+            /*@@*/ await WaitForCompletionAsync(releaseOperation);
```

Comment:
@pavelprystinka, some of the tests were taking nearly a minute in playback mode. I added this function: https://github.com/Azure/azure-sdk-for-net/pull/18037/files#diff-5636043a2134cb7dcd123d58c6625cf1dd400c6daa1ad7879fe08e6a97e8be2cR245 to make sure the delay only happens in record and live mode

Reply:
💗